.github/etc/dictionary.txt

```diff
@@ -1,2 +1,5 @@
 capricornis
```

Comment:
The dictionary.txt file appears to be a list of words. It seems that the words kubeconform, textlint, and hadolint have been added. Please verify that these are legitimate words and should be included in the dictionary.